packages/recozik-web/src/recozik_web/app.py

```
@@ -73,8 +76,11 @@ def _token_from_websocket(websocket: WebSocket) -> str | None:
     try:
         jar = SimpleCookie()
         jar.load(cookie_header)
-        cookie = jar.get("recozik_token")
-        return cookie.value if cookie and cookie.value else None
+        for name in ("recozik_token", ACCESS_COOKIE):
+            cookie = jar.get(name)
+            if cookie and cookie.value:
+                return cookie.value
+        return None
```

Ruff (0.14.5)

Consider moving this statement to an `else` block

(TRY300)